### ISSUE TITLE
Add a "Date Approved" field to Techniques and Failures

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#183]
I think we have a problem introducing failures that we will have to address in WCAG.NEXT. I would like to propose a solution.
## Problem

WCAG was created to be an ever green document. The SCs are not technology dependent on purpose to allow the standard to remain relevant in a technologically changing world. Non normative techniques and failures, can be created to address new realities that we see on the ground as the web develops. This has happened for techniques, but _**not**_ failures. We have created about 150 new techniques since 2008, and only **3** (three) new failures (all of which were basically administrative).

It is not from a lack of failure proposals, there have been plenty in 8 years. However, it is almost impossible to gain consensus on a failure, because there is always some concern about tightening things up, for various reasons, some of which I would agree with in some situations. Here are the main reasons its hard to pass a failure:
1.  Concern that it changes the requirements of WCAG
2. Concern that there is a ****perceived**** change to WCAG
3. Concern that pages that once passed will not pass after a new common failure is introduced.
## Solution

I'd like to propose an "Approved date" field, to techniques and failures, which would be populated when we gained consensus on a technique or failure. This will give jurisdictions a tool to exempt failures that were created after a site was built.

**Comment by mraccess77** [comment]
I'm ok with a date field -- but I don't think we should use the word "approved".  I believe I echo Gregg's comments that a better term would be something like "date documented" or something similar as any failure is and was always a failure.

**Comment by joshueoconnor** [comment]
@jnurthen suggested Date reviewed - this got some +1s

**Comment by joshueoconnor** [comment]
http://www.w3.org/2016/05/17-wai-wcag-minutes.html

**Comment by fstrr** [comment]
Closing this issue due to inactivity. If it needs to be re-opened, please do so.
